deploy/gcp/outputs.tf

```diff
@@ -0,0 +1,19 @@
+output "region" {
+  value = "${var.GCP_REGION}"
+}
+
+output "cluster_id" {
+  value = "${google_container_cluster.cluster.id}"
+}
+
+output "kubeconfig_file" {
+  value = "${local.kubeconfig}"
+}
+
+output "tidb_version" {
+  value = "${var.tidb_version}"
+}
+
+output "tidb_ilb_ip" {
+  value = "${data.external.tidb_ilb_ip.result["ip"]}"
+}
```

Comment:
It will be more clear for user if we expose `cluster_name` in output (user will use this value in many places according to the README)